website/content/docs/auth/aws.mdx

```diff
@@ -674,6 +674,10 @@ If the region is specified as `auto`, the Vault CLI will determine the region ba
 on standard AWS credentials precedence as described earlier. Whichever method is used,
 be sure the designated region corresponds to that of the STS endpoint you're using.
 
+-> **Note on GovCloud:** If you are making use of AWS GovCloud and setting the `sts_endpoint`
+and `sts_region` role parameters to `us-gov-west-1` / `us-gov-east-1` then you should include
```

Comment:
if this is something they are required to do, I recommend saying something like, "you must include..."